packages/react-vapor/rollup.config.js

```diff
@@ -1,16 +1,18 @@
-import path from 'path';
 import commonjs from '@rollup/plugin-commonjs';
 import inject from '@rollup/plugin-inject';
 import resolve from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
+import path from 'path';
 import postcss from 'rollup-plugin-postcss';
 import scssVariable from 'rollup-plugin-sass-variables';
+import {terser} from 'rollup-plugin-terser';
 import typescript from 'rollup-plugin-typescript2';
 import keysTransformer from 'ts-transformer-keys/transformer';
 
 export default [
     {format: 'es', file: 'dist/react-vapor.esm.js'},
     {format: 'umd', file: 'dist/react-vapor.js'},
+    {format: 'umd', file: 'dist/react-vapor.min.js'},
```

Comment:
As discussed in Slack I would only add this format when we compile on Jenkins to avoid the impact of another compilation locally

Reply:
good idea ! :+1: